src/Makefile

```diff
@@ -0,0 +1,16 @@
+BINDIR = ${DESTDIR}/usr/bin
+PKG_CONFIG = pkg-config
+PKG_NAMES = ssu-sysinfo
+PKG_LIBS = $(shell $(PKG_CONFIG) --libs $(PKG_NAMES))
+PKG_CFLAGS = $(shell $(PKG_CONFIG) --cflags $(PKG_NAMES))
+LIBS += $(PKG_LIBS)
+CFLAGS += $(PKG_CFLAGS)
+
+sample-app-c: sample-app-c.c
```

Comment:
maybe add the 'all' or 'first' target or how it is usually called?

Reply:
Kind of feels redundant, as the that would be just "all: sample-app-c".

Reply:
yup, it feels redundant in this little example, but this little example is here to show the big world in a nutshell, so to me it would make sense to follow the big world conventions and add some well known name for the default target :)